mw-webapp/src/component/input/Input.tsx

```diff
@@ -0,0 +1,39 @@
+import {FC, HTMLInputTypeAttribute} from "react";
+import cn from "classnames";
+import styles from "./Input.module.scss";
+
+
+interface InputProps {
+  type?: HTMLInputTypeAttribute;
+  placeholder?: string;
+  className?: string | Array<string>;
+  inputMode?: "none" | "text" | "tel" | "url" | "email" | "numeric" | "decimal" | "search" | undefined;
+  disabled?: boolean;
+  required?: boolean;
+}
+
+export const Input: FC<InputProps> = ({
+  type = "text",
+  placeholder = "",
+  className = "",
+  disabled = false,
+  required = false,
+  inputMode,
+}) => {
+  return (
+    <div className={styles.input__wrapper}>
+      <label
+        className={styles["visually-hidden"]}
+      >
+        {placeholder}
```

Comment:
if it is hidden why do you need to render a placeholder inside?

Reply:
https://www.w3.org/WAI/tutorials/forms/labels/
![image](https://github.com/tritonJS826/masters-way/assets/32575876/5610c5cd-c5a5-4278-9714-2431fd616517)